addons/l10n_account_edi_ubl_cii_tests/tests/test_xml_ubl_nl.py

```diff
@@ -198,6 +209,34 @@ def test_export_import_refund(self):
         self.assertEqual(xml_filename[-10:], "nlcius.xml")
         self._assert_imported_invoice_from_etree(refund, xml_etree, xml_filename)
 
+    def test_export_fixed_tax_nlcius_and_peppol(self):
+        """
+        Ensure that an invoice containing a product with a fixed tax posted to a journal with the peppol and nlcius edi
+            tags generates edi documents with accurate LineExtensionAmount values
+        """
+        self.journal.edi_format_ids = [
+            Command.link(self.env.ref('account_edi_ubl_cii.ubl_bis3').id)
+        ]
+
+        self.tax_10_fixed.sequence = 1
+        invoice = self._generate_move(
+            self.partner_1, self.partner_2,
+            move_type='out_invoice',
+            invoice_line_ids=[{
+                'name': 'product costing 50.0',
+                'quantity': 1,
+                'price_unit': 50.0,
+                'tax_ids': [Command.set([self.tax_10_fixed.id, self.tax_7_purchase.id])]
+            }]
+        )
+
+        peppol_amount, nlcius_amount = [
+            etree.fromstring(xml.raw).find('.//{*}LegalMonetaryTotal/{*}LineExtensionAmount').text
+            for xml in invoice.edi_document_ids[1:].attachment_id
+        ]
+
+        self.assertEqual((peppol_amount, nlcius_amount), ('60.00', '60.00'))
```

Comment:
The order of of edi document might not always be the same, what about using:
```suggestion
        amounts = [
            etree.fromstring(doc.attachment_id.raw).find('.//{*}LegalMonetaryTotal/{*}LineExtensionAmount').text
            for doc in invoice.edi_document_ids.filtered(lambda d: d.edi_format_id.code in ('ubl_bis3', 'nlcius_1'))
        ]

        self.assertEqual(amounts, ['60.00', '60.00'])
```